awxkit/awxkit/cli/__init__.py

```
@@ -60,7 +58,7 @@ def run(stdout=sys.stdout, stderr=sys.stderr, argv=[]):
             sys.stdout.write(e.__class__.__name__)
             print('')
         sys.exit(1)
-    except Exception as e:
+    except Exception as e:  # noqa: F841
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)